files/SZWESAMP/ZWEIMVS2

```diff
@@ -0,0 +1,45 @@
+//ZWEIMVS2 JOB
+//*
+//* This program and the accompanying materials are made available
+//* under the terms of the Eclipse Public License v2.0 which
+//* accompanies this distribution, and is available at
+//* https://www.eclipse.org/legal/epl-v20.html
+//*
+//* SPDX-License-Identifier: EPL-2.0
+//*
+//* Copyright Contributors to the Zowe Project. 2020, 2020
+//*
+//*********************************************************************
+//*
+//* This job is used to create the APF load library for an instance
+//* Of Zowe. It is not needed if your choosen value of
+//* 'zowe.setup.dataset.authLoadlib' is equal to
+//* 'zowe.setup.prefix' + 'SZWELOAD'.
+//*
+//* When running this job, you should also run ZWEIMVS
+//*
+//*********************************************************************
+//MKPDSE EXEC PGM=IKJEFT01
+//SYSTSPRT DD SYSOUT=A
+//SYSTSIN DD *
+ALLOC NEW DA('{zowe.setup.dataset.authLoadlib}') +
+dsntype(library) dsorg(po) recfm(u) lrecl(0) +
+blksize(32760) unit(sysallda) space(30,15) tracks
+//*
+//AUTHCPY EXEC PGM=BPXBATCH
+//BPXPRINT DD SYSOUT=*
+//STDOUT   DD SYSOUT=*
+//STDERR   DD SYSOUT=*
+//STDPARM DD *
+SH cd "{zowe.runtimeDirectory}" &&
+cd components/zss &&
+cp -X LOADLIB/ZWESIS01
+"//'{zowe.setup.dataset.authLoadlib}(ZWESIS01)'" &&
+cp -X LOADLIB/ZWESAUX
+"//'{zowe.setup.dataset.authLoadlib}(ZWESAUX)'" &&
+cp -X LOADLIB/ZWESISDL
+"//'{zowe.setup.dataset.authLoadlib}(ZWESISDL)'" &&
+cd ../launcher/bin &&
+cp -X zowe_launcher
+"//'{zowe.setup.dataset.authLoadlib}(ZWELNCH)'"
+/*
```

Comment:
This will fail for
* `zowe.runtimeDirectory` with `$`, which will not be probably used by customers, but it is legal character for unix path
  * This can be easily done with single apostrophes
* `zowe.setup.dataset.authLoadlib` containing `$`, which could be used
  * I did not find better way how to treat dataset with dollar sign than in the example below
  * You don't need to check/know which chars are used for ds and it will work:
```
SH cd '{zowe.runtimeDirectory}' &&
cd components/zss &&
ZWE__AUTHLOADLIB='{zowe.setup.dataset.authLoadlib}' &&
cp -X LOADLIB/ZWESIS01
"//'$ZWE__AUTHLOADLIB(ZWESIS01)'" &&
cp -X LOADLIB/ZWESAUX
"//'$ZWE__AUTHLOADLIB(ZWESAUX)'" &&
cp -X LOADLIB/ZWESISDL
"//'$ZWE__AUTHLOADLIB(ZWESISDL)'" &&
cd ../launcher/bin &&
cp -X zowe_launcher
"//'$ZWE__AUTHLOADLIB(ZWELNCH)'"
```

Reply:
thanks, excellent idea and will do.

Reply:
Done [here](https://github.com/zowe/zowe-install-packaging/pull/3718/commits/a47703f0018d7061467eb241370c6202ea6c18bf).